datadog_checks_dev/tests/test_docker.py

```diff
@@ -40,3 +44,60 @@ def test_compose_file(self):
             assert compose_file_active(compose_file) is False
         finally:
             run_command(['docker', 'compose', '-f', compose_file, 'down'], capture=True)
+
+    @pytest.mark.parametrize(
+        "attempts,expected_call_count",
+        [
+            [
+                None,
+                1,
+            ],
+            [
+                0,
+                1,
+            ],
+            [
+                1,
+                1,
+            ],
+            [
+                3,
+                3,
+            ],
+        ],
```

Comment:
I would try to format this in fewer lines, it's a little hard to read like this and it takes a lot of unnecessary vertical space. I think if you remove the dangling `,` (at the end of each list) and re-run the formatter it should come up with a more sensible layout.

Another small nit / suggestion: I find that using a list for the list of cases and a tuple for the list of args tends to make these easier to read, too, but YMMV.

Reply:
It should be better now, Thanks for the suggestion. 

Good idea for the tuple vs list args